lighthouse-core/test/aggregator/aggregate.js

```diff
@@ -125,16 +125,16 @@ describe('Aggregate', () => {
       'Cannot remap: test already exists');
   });
 
-  it('returns a weight of zero for undefined inputs', () => {
-    return assert.equal(Aggregate._convertToWeight(), 0);
+  it('throws for undefined inputs', () => {
+    return assert.throws(_ => Aggregate._convertToWeight(), 0);
```

Comment:
Maybe also add a test of `Aggregate.compare()` with an expected value missing a `rawValue` property?